server/db/populate-db.sql

```diff
@@ -13,6 +13,41 @@ INSERT INTO languages (code, name) VALUES ('be','Bengali');
+De todas las numerosas palabras que figuran en dicha comunicación,
+tan sólo un párrafo breve está dedicado a esa cuestión,');
+
+INSERT INTO documents (name, from_language_code, to_language_code, status, submission_date, due_date, owner_id, format, content) values ('Bank Statement', 'es','fr', 'Waiting', '2019-01-20','2019-02-20','db5b2e53-acd2-4af0-bd91-8554d9dc09ef','Text', E'Wikipedia define un extracto bancario como "un resumen de las transacciones financieras que tienen
+ocurrió durante un período determinado en una cuenta bancaria en poder de una persona o estado de cuenta comercial con una institución financiera. ”Consiste en transacciones como débito y crédito, retiros y depósitos. Los extractos bancarios generalmente se envían por correo en su formato tradicional de papel. Algunas personas aún eligen este método para que puedan verificar y ver literalmente qué actividades se han realizado a través de sus cuentas bancarias. Sin embargo, debido a la aparición de nuevas tecnologías, los extractos bancarios han evolucionado hasta convertirse en documentos que ahora se envían por correo electrónico y otros métodos digitales para enviarlos.
+El dinero es un factor importante en nuestras actividades cotidianas. Compra la comida que comemos, nos lleva a transportar de un lugar a otro y obtiene las cosas que podrían hacernos felices. Sin embargo, cuando ocurren estos problemas, lo mejor que puede hacer es comunicarse con su banco. Puede ir a su sucursal más cercana o llamar a su línea directa de atención al cliente y solicitar algunas aclaraciones y explicaciones de tales eventos.
+Té
+');
+
+
+INSERT INTO documents (name, from_language_code, to_language_code, status, submission_date, due_date, owner_id, format, content) values ('Tax Form', 'en','es', 'Waiting', '2019-01-20','2019-02-20','db5b2e53-acd2-4af0-bd91-8554d9dc09ef','Text', E'Filling out a tax form is about as much fun as owing taxes to Uncle Sam. The complexity of the task intensifies as your tax life becomes more complicated.
+That’s why you should use the simplest tax return form you can, especially if you’re still filling out your forms by hand.
+
+But choose carefully. There are three personal income tax forms — 1040, 1040A and 1040EZ — with each designed to get the appropriate amount of your money to the IRS. Differences in the forms, however, could cost you if you’re not paying attention.
+
+The EZ is the shortest and simplest form, Form 1040A is a bit more complex and the long Form 1040 is the most detailed and potentially difficult. But even if your tax life is simple and straightforward, it might be worthwhile to investigate the other two forms. Why? Generally, the longer the form, the more opportunities for tax breaks.
+
+RATE SEARCH: Shopping for a mortgage? Compare mortgage rates today at Bankrate.com.');
+
```

Comment:
Can you modify the following things?

1. Change the due dates of every document so the day of the presentation (9th of October) most of the documents are not Overdue?
2. Show one overdue, and one overdue in 2 days.
3. Change the name of the user to Vicky (she will be the user on the presentation, right?)
4. Change the owner_id of all the documents so they belong to Vicky
5. Can you create another user on the db (Loic, for example)
6. Can you add 5 more documents and assign them to Loic?

Reply:
ok i'll do as requested